src/widgets/AppWindow.js

```diff
@@ -90,8 +91,11 @@ export default class AppWindow extends createjs.Container {
       },
     });
     this._headerArea.accessible.addChild(headerText);
+    this._headerArea.accessible.title = 'Createjs Accessibility Test APP';
+    this._headerArea.accessible.spellcheck = true;
```

Comment:
How about moving the spellcheck case to the multi-line text input in the form test case?

Reply:
updated